projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cpq/cpq-configuration.ccv2-e2e-spec.ts

```diff
@@ -0,0 +1,589 @@
+        CHKBOX_PROD,
+        '2x($100.00) + $200.00',
+        ATTR_CAM_MC,
+        VAL_CAM_MC_128
+      );
+
+      configurationCpq.selectProductCard(CHKBOX, ATTR_CAM_LEN, VAL_CAM_LEN_SI);
+      configurationCpq.selectProductCard(CHKBOX, ATTR_CAM_LEN, VAL_CAM_LEN_NI);
+      configurationCpq.checkPrice(
+        CHKBOX_PROD,
+        '$800.00',
+        ATTR_CAM_LEN,
+        VAL_CAM_LEN_SI
+      );
+      configurationCpq.checkPrice(
+        CHKBOX_PROD,
+        '$700.00',
+        ATTR_CAM_LEN,
+        VAL_CAM_LEN_NI
+      );
+
+      configuration.clickOnNextBtn(GRP_CAM_ACC);
+      configurationCpq.deSelectProductCard(
+        RADGRP,
+        ATTR_CAM_BAG,
+        VAL_CAM_BAG_LP
+      );
+
+      configuration.clickOnNextBtn(GRP_CAM_IAW);
+      configurationCpq.selectAttributeAndWait(
+        ATTR_CAM_PROF,
+        RADGRP,
+        VAL_CAM_PROF_Y
+      );
+      //wait for this option to disappear
+      configuration.checkAttrValueNotDisplayed(
+        ATTR_CAM_INS,
+        DDLB_PROD,
+        VAL_CB_INS_Y2
+      );
+
+      configurationCpq.selectProductCard(DDLB, ATTR_CAM_INS, VAL_CB_INS_P4);
+      configurationCpq.checkPrice(
+        DDLB_PROD,
+        '$600.00',
+        ATTR_CAM_INS,
+        VAL_CB_INS_P4
+      );
+      configuration.navigateToOverviewPage();
+
+      configurationOverview.checkConfigOverviewPageDisplayed();
+      configurationOverviewCpq.checkGroupHeaderDisplayed(GRP_CAM_MAIN, 0);
+      configurationOverviewCpq.checkGroupHeaderNotDisplayed(GRP_CAM_ACC);
+      configurationOverviewCpq.checkGroupHeaderDisplayed(GRP_CAM_IAW, 1);
+
+      type ovContentType = {
+        name?: string;
+        value: string;
+        type: 'product' | 'simple';
+        price?: string;
+      };
+      const ovContent: ovContentType[] = [
+        {
+          name: 'Camera Body',
+          value: 'Nikon D850',
+          type: 'product',
+          price: '+ $1,500.00',
+        },
+        {
+          name: 'Memory Card',
+          value: 'SanDisk Extreme Pro',
+          type: 'product',
+          price: '2x($100.00) + $200.00',
+        },
+        {
+          name: 'Lenses',
+          value: 'Sigma 85mm F1.4 DG HS',
+          type: 'product',
+          price: '+ $800.00',
+        },
+        { value: 'Nikon AF-P DX NIKKOR', type: 'product', price: '+ $700.00' },
+        {
+          name: 'professional photographer?',
+          value: 'yes',
+          type: 'simple',
+        },
+        {
+          name: 'Insurance',
+          value: 'Pro 4 years',
+          type: 'product',
+          price: '+ $600.00',
+        },
+      ];
+      ovContent.forEach((line, idx) => {
+        configurationOverviewCpq.checkAttrDisplayed(line.name, line.value, idx);
+        configurationOverviewCpq.checkAttrPriceDisplayed(line.price, idx);
+        configurationOverviewCpq.checkAttrType(line.type, idx);
+      });
+    });
+  });
+
+  /*
+  describe('Configuration Process', () => {
```

Comment:
What is the reason for commenting it out? 

From my understanding, I believe it was commented out since it did not work ootb on the p3 env on ccv2? Therefore, I'm assuming this new file was created to just comment this `describe` out, which is the only difference

Maybe it's best to fix the e2e test case instead?

---
For a general information on why there is a new ccv2 folder

- Giancarlo created them in favor for getting them quick and stable as some people were waiting for results for Spartacus testing on ccv2

Reply:
I will delete the comment.
The test is in the vendor folder, because it is dependent on a third party system. Therefor we do not want them to be executed when ccv2-b2c or ccv2-b2b tests are executed.

Reply:
I understand you don't wan to execute it for b2c/b2b test suite

my main question was more towards, why is it commented out? I believe the file is exactly the same as the other file in the same folder, but was just removed because it didn't work well with ccv2 p3. If it does not work well on ccv2 p3, maybe it's best to update that test case to adhere to what p3 has.

Reply:
The problem is more of a user/test data type.
We use specific users in our own infrastructure tests, which are not available on P3.